### PR TITLE
Make jvarness an alum

### PR DESCRIPTION
**Pull request by jvarness**
Hey everyone,

I'm sorry for radio silence over the last... Well, it's been a very long time. I have not had the spare time to commit to being a mentor or a maintainer on Exercism. As such, I wanted to mark myself as an alum.

Do know that I'm alive and well and that I very much would like to contribute to the track again in the future (time permitting :) )

### CODE REVIEW

File: config/maintainers.json [hunk 1]
```diff
@@ -43,13 +43,13 @@
...
       "github_username": "jvarness",
-      "alumnus": false,
+      "alumnus": true,
       "show_on_website": true,
       "name": "Jake Varness",
       "link_text": "dev.to",
-      "link_url": "https://dev.to/jvarness",
+      "link_url": "https://jvarness.blog/",
       "avatar_url": "https://avatars1.githubusercontent.com/u/6510879?s=460\u0026v=460",
-      "bio": "Team Lead and Senior Software Engineer at Cerner Corporation, husband, father, nerd"
```
Comment (devkabiir) [inline config/maintainers.json:52]: Might not be required to remove your bio to mark you as alumnus unless you want to.

**Review by devkabiir — APPROVED** [review]
Thanks for all of your contributions. You're welcome to join again anytime. 🙂

File: config/maintainers.json (hunk 1 above)
Comment (jvarness) [reply config/maintainers.json:52]: I would prefer to for now :)

**Review by devkabiir — APPROVED** [review]

**Comment by devkabiir** [comment]
Weird, I can't merge it. It says I don't have write access. 🤷‍♂️

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
> Weird, I can't merge it. It says I don't have write access.

You should have that now. The `config/maintainers.json` file is protected in the [CODEOWNERS](https://github.com/exercism/dart/blob/main/.github/CODEOWNERS#L2) file, and thus requires an Exercism admin to approve.

**Comment by jvarness** [comment]
> Thanks Jake. Hopefully you'll be back soon! Good luck with whatever you're doing :)

Thanks so much Jeremy! Exercism is a really cool product, and one that I hope to contribute to again in the future!

**Pull request merged**